content/en/docs/setup/other_config/security/authorization/github-teams/index.md

```diff
@@ -6,7 +6,74 @@ description: Spinnaker supports using GitHub teams for authorization. Roles from
 
 
 
-## Personal Access Token
+## GitHub App Authentication (Recommended)
```

Comment:
If we can, "available in releases after 2025.4.0".  We'll likely look at merging this after this gets into an official release.